hw/ip/rv_timer/data/rv_timer.hjson

```diff
@@ -39,10 +46,7 @@
         swaccess: "rw",
         hwaccess: "hro",
         fields: [
-          { bits: "0", name: "active",
-            desc: "If 1, timer operates",
-            tags: [// prevent timer from being enabled
-                   "excl:CsrNonInitTests:CsrExclWrite"] }
```

Comment:
@msfschaffner I think your changes removed some original exclusion tags from this hjson file.
Is this intended, could you add them back if not?

Reply:
ah I know what is going on here: 
I was editing the template `rv_timer.hjson.tpl` and regenerated this file. 
turns out these exclusions have been added to the `hjson` directly - and not the template.